src/core/hle/D3D8/XbVertexBuffer.cpp

```diff
@@ -69,6 +69,44 @@ uint32_t GetPixelContainerWidth(xbox::X_D3DPixelContainer* pPixelContainer);
 uint32_t GetPixelContainerHeight(xbox::X_D3DPixelContainer* pPixelContainer);
 void ApplyXboxMultiSampleOffsetAndScale(float& x, float& y);
 
+_D3DIVB::_D3DIVB()
+{
+	Position.x = 0.0f;
+	Position.y = 0.0f;
+	Position.z = 0.0f;
+	Rhw = 0.0f;
+	std::fill(std::begin(Blend), std::end(Blend), 0.0f);
+	Normal.x = 0.0f;
+	Normal.y = 0.0f;
+	Normal.z = 0.0f;
+	Diffuse = 0u;
+	Specular = 0u;
+	Fog = 0.0f;
+	BackDiffuse = 0u;
+	BackSpecular = 0u;
+	std::fill_n(std::begin(TexCoord), 4, D3DXVECTOR4{ 0.0f , 0.0f, 0.0f, 0.0f });
```

Comment:
`TexCoord` is a normal array just like `Blend`, so can just use a normal fill
```suggestion
	std::fill(std::begin(TexCoord), std::end(TexCoord), D3DXVECTOR4{ 0.0f , 0.0f, 0.0f, 0.0f });
```